../epiphany-42.1/lib/safe-browsing/ephy-gsb-service.c:121:45: warning: format specifies type 'long' but the argument has type 'gint64' (aka 'long long') [-Wformat]

../epiphany-42.1/lib/safe-browsing/ephy-gsb-service.c:170:48: warning: format specifies type 'long' but the argument has type 'gint64' (aka 'long long') [-Wformat]

../epiphany-42.1/lib/safe-browsing/ephy-gsb-service.c:591:10: warning: format specifies type 'long' but the argument has type 'long long' [-Wformat]

../epiphany-42.1/lib/safe-browsing/ephy-gsb-service.c:597:10: warning: format specifies type 'long' but the argument has type 'long long' [-Wformat]

Index: lib/safe-browsing/ephy-gsb-service.c
--- lib/safe-browsing/ephy-gsb-service.c.orig
+++ lib/safe-browsing/ephy-gsb-service.c
@@ -118,7 +118,7 @@ ephy_gsb_service_update_back_off_mode (EphyGSBService 
   ephy_gsb_storage_set_metadata (self->storage, "back_off_exit_time", self->back_off_exit_time);
   ephy_gsb_storage_set_metadata (self->storage, "back_off_num_fails", self->back_off_num_fails);
 
-  LOG ("Set back-off mode for %ld seconds", duration);
+  LOG ("Set back-off mode for %lld seconds", duration);
 }
 
 static inline void
@@ -167,7 +167,7 @@ ephy_gsb_service_schedule_update (EphyGSBService *self
                                            self);
   g_source_set_name_by_id (self->source_id, "[epiphany] gsb_service_update");
 
-  LOG ("Next update scheduled in %ld seconds", interval);
+  LOG ("Next update scheduled in %lld seconds", interval);
 }
 
 static gboolean
@@ -587,13 +587,13 @@ ephy_gsb_service_update_full_hashes_in_thread (UpdateF
   g_mutex_lock (&data->mutex);
 
   if (self->next_full_hashes_time > CURRENT_TIME) {
-    LOG ("Cannot send fullHashes:find request. Requests are restricted for %ld seconds",
+    LOG ("Cannot send fullHashes:find request. Requests are restricted for %lld seconds",
          self->next_full_hashes_time - CURRENT_TIME);
     goto unlock;
   }
 
   if (ephy_gsb_service_is_back_off_mode (self)) {
-    LOG ("Cannot send fullHashes:find request. Back-off mode is enabled for %ld seconds",
+    LOG ("Cannot send fullHashes:find request. Back-off mode is enabled for %lld seconds",
          self->back_off_exit_time - CURRENT_TIME);
     goto unlock;
   }
